### PR TITLE
fix(flow): failed to create a ticket which manual strategy in individual space

### LINKED ISSUE #2510: [Bug]: Error message when selecting manual execution when creating a personal space task
### ODC version

odc430

### OB version

*

### What happened?

image

### What did you expect to happen?

Execution successful

### How can we reproduce it (as minimally and precisely as possible)?

Personal space creation task, select manual execution

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by yiminpeng**
#### What type of PR is this?

type-bug

#### What this PR does / why we need it:

Fixed failed to create a ticket which manual strategy in  individual space
errorMsg: “WaitExecExpireIntervalSeconds is too large”

#### Which issue(s) this PR fixes:
Fixes #2510 

#### Additional documentation e.g., usage docs, etc.:

### CODE REVIEW

**Review by MarkPotato777 — COMMENTED** [review]
We better not involve updating submodule if not related to your issue.

File: server/odc-service/src/main/java/com/oceanbase/odc/service/flow/model/ExecutionStrategyConfig.java [hunk 1]
```diff
@@ -61,7 +61,8 @@ public static ExecutionStrategyConfig autoStrategy() {
     public static ExecutionStrategyConfig manualStrategy(long expireIntervalTime, @NonNull TimeUnit timeUnit) {
         PreConditions.notNegative(expireIntervalTime, "WaitExecExpireIntervalSeconds");
         int expireIntervalTimeSeconds = (int) TimeUnit.SECONDS.convert(expireIntervalTime, timeUnit);
-        Verify.verify(expireIntervalTime > 0, "WaitExecExpireIntervalSeconds is too large " + expireIntervalTime);
+        // if the expression is false, it means that the message will be printed in the console
```
[the remark this answers is not in the document]
Comment (yiminpeng) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/flow/model/ExecutionStrategyConfig.java]: get

**Review by MarkPotato777 — APPROVED** [review]
LGTM

File: server/odc-service/src/main/java/com/oceanbase/odc/service/flow/model/ExecutionStrategyConfig.java [hunk 2]
```diff
@@ -61,7 +61,6 @@ public static ExecutionStrategyConfig autoStrategy() {
     public static ExecutionStrategyConfig manualStrategy(long expireIntervalTime, @NonNull TimeUnit timeUnit) {
         PreConditions.notNegative(expireIntervalTime, "WaitExecExpireIntervalSeconds");
         int expireIntervalTimeSeconds = (int) TimeUnit.SECONDS.convert(expireIntervalTime, timeUnit);
-        Verify.verify(expireIntervalTime > 0, "WaitExecExpireIntervalSeconds is too large " + expireIntervalTime);
```
Comment (yhilmare) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/flow/model/ExecutionStrategyConfig.java:64]: `>=0` will be better

**Review by yhilmare — APPROVED** [review]
lgtm

**Pull request merged**